xontrib/fzf-widgets.xsh

```diff
@@ -41,6 +41,7 @@ def fzf_insert_history(event):
 
     if choice:
         event.current_buffer.text = choice
+        event.current_buffer.cursor_position = len(choice)
```

Comment:
Could this please be `cursor_position += len(choice)` instead?

Reply:
Never mind. I missed the fact that the buffer text is being replaced by the choice.